spec/ics-027-interchain-account/README.md

```diff
@@ -0,0 +1,292 @@
+---
+ics: 27  
+title: Interchain Account  
+stage: Draft  
+category: IBC/TAO  
+author: Tony Yun <yunjh1994@everett.zone>, Dogemos <josh@tendermint.com>  
+created: 2019-08-01  
+modified: 2019-11-08  
+---  
+
+## Synopsis  
+
+This standard document specifies packet data structure, state machine handling logic, and encoding details for the account management system over an IBC channel between separate chains.  
+
+### Motivation  
+
+On Ethereum, there are two types of accounts: externally owned accounts, controlled by private keys, and contract accounts, controlled by their contract code [[ref](https://github.com/ethereum/wiki/wiki/White-Paper)]. Similar to Ethereum's CA (contract accounts), interchain accounts are managed by another chain while retaining all the capabilities of a normal account (i.e. stake, send, vote, etc). While an Ethereum CA's contract logic is performed within Ethereum's EVM, interchain accounts are managed by another chain via IBC in a way such that the owner of the account retains full control over how it behaves.  
+
+### Definitions  
+
+The IBC handler interface & IBC relayer module interface are as defined in [ICS 25](https://github.com/cosmos/ics/blob/bez/15-ics-cosmos-signed-messages/spec/ics-025-handler-interface) and [ICS 26](https://github.com/cosmos/ics/blob/bez/15-ics-cosmos-signed-messages/spec/ics-026-relayer-module), respectively.  
+
+### Desired Properties  
+
+- Permissionless  
+- Fault containment: Interchain account must follow rules of its host chain, even in times of Byzantine behavior by the counterparty chain (the chain that manages the account)  
+- The chain that controls the account must process the results asynchronously and according to the chain's logic.  
+- Sending and receiving transactions will be processed in an ordered manner.  
```

Comment:
As in, first in first out? Be specific (there could be many kinds of orderings).